### PR TITLE
Don't add empty elements. Fix styles

### PR DESCRIPTION
**Pull request by terotic**
* Do not show header or lead elements on closure info box if they have no content
* Fix closure info styles accordingly

### CODE REVIEW

File: src/components/Section.js [hunk 1]
```diff
@@ -169,7 +172,7 @@ export class Section extends React.Component {
       {titleDiv}
       <div className="section-content">
         {imageList}
-        {section.type !== "main" ?
+        {section.type !== "main" && section.abstract ?
```
Comment (Rikuoja) [inline src/components/Section.js]: The problem here is that section.abstract is always present even if empty (an empty object evaluates as true due to ES6 silliness). `getAttr(section.abstract, language)` will return `undefined` if the object is empty, that should be good :D

Comment (akx) [inline src/components/Section.js]: [`_.isEmpty`](https://lodash.com/docs/#isEmpty).

File: src/components/Section.js [hunk 2]
```diff
@@ -57,11 +57,14 @@ export class Section extends React.Component {
       return null;
     }
     if (section.type === "closure-info") {
-      return (
-        <h3 className="section-title">
-          {getAttr(section.title, language)}
-        </h3>
-      );
+      if (section.title.isString) {
```
Comment (akx) [inline src/components/Section.js]: I don't think these objects have `isString`...

File: src/components/Section.js [hunk 3]
```diff
@@ -169,7 +172,7 @@ export class Section extends React.Component {
       {titleDiv}
       <div className="section-content">
         {imageList}
-        {section.type !== "main" ?
+        {section.type !== "main" && section.abstract.isEmpty ?
```
Comment (akx) [inline src/components/Section.js]: ... nor do I think they have `isEmpty`.

You'll need to
```javascript
import isEmpty from 'lodash/isEmpty';
import isString from 'lodash/isString';
```
and
```javascript
if(isEmpty(...))
```

**Pull request merged**